docs/development/decision-records/2025-02-12-dsp-downward-compatibility/README.md

```diff
@@ -0,0 +1,20 @@
+# Handling of downward compatibility for a DSP version
+
+## Decision
+
+DSP assets, like the catalog, which are forwarded directly to a caller through the management api, have to be kept downward compatible as long as the DSP version through which the asset is retrieved is supported. There is no compatibility requirement between two different DSP versions concerning such assets.
+
+Relevant assets are:
+- Catalog returned by...
+- ...
+
+## Rationale
+
+DSP assets which are directly received by a caller of a management api endpoint is data that is transfered between two dataspace participants and creates a direct relationship between the calling application and the provider connector. As this is the case, the breaking change requirements of Catena-X has to be fulfilled. The requirement states that a consumer can continuously consume data from the provider without the need to be forced to update his system due to a change of the provider service stack. This requires that such assets must not change in a way, that existing properties vanish from the returned data.
```

Comment:
```suggestion
DSP datasets which are obtained by a consumer third party app through the management api endpoint create a direct relationship between the calling application and the provider connector. As this is the case, the breaking change requirements of Catena-X has to be fulfilled. The requirement states that a consumer can continuously consume data from the provider without the need to be forced to update his system due to a change of the provider service stack. This requires that such assets must not change in a way, that existing properties vanish from the returned data.
```

this need to be rephrased:
> As this is the case
which case?

plus, a link to the "breaking change requirements of Catena-X" will be necessary